.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,4 @@
+command:
```

Comment:
I'm missing a folder check for the `data` mounted volume. Additionally, is there any conf file where we could check any of the set params on `values.yaml`?

Reply:
I've added a check for the data directory but it's empty. The cfg file is also mounted and the test is checking the contents of that as well. (for a value that is set in the values.yaml)